### DISCUSSION TITLE
Adjust machine resources

### DISCUSSION THREAD

**Discussion by sj14** [podman-desktop/podman-desktop#808] [Q&A] [answered]
Hi, is it possible to adjust a machine (e.g. CPUs and RAM)? As far as I can see, I can only create new machines or delete machines, but not edit them.

**Comment by benoitf** [comment, marked as answer]
Hi @sj14
At first, podman was not allowing but since Podman 4.2 it's possible but not yet handle on Podman Desktop side.
But yes, this is definitely something that we want to make it work.

For now there is a workaround using `podman machine set --cpus <value>` or `podman machine set --memory <value>` but on command line
